Userland/Utilities/image.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023, Nico Weber <thakis@chromium.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibCore/MappedFile.h>
+#include <LibGfx/BMPWriter.h>
+#include <LibGfx/ImageDecoder.h>
+#include <LibGfx/PNGWriter.h>
+#include <LibGfx/QOIWriter.h>
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    Core::ArgsParser args_parser;
+
+    StringView in_path;
+    args_parser.add_positional_argument(in_path, "Path to input image file", "FILE");
+
+    StringView out_path;
+    args_parser.add_option(out_path, "Path to output image file", "output", 'o', "FILE");
```

Comment:
having one as a positional argument and the other as an option seems somewhat odd, why not `image <input> <output>`?

Reply:
Using `-o` for output seems pretty standard. For `<input> <output>` it's never quite clear which one is which too.

Reply:
And you could imagine things like `-o .png` or `--to-format=png` or similar which writes to the same filename as the input with just the extension replaced with ".png". That's weirder to do if it's `<input> <output>`.